core/posts.go

```diff
@@ -21,19 +21,106 @@ import (
 
 // Constants for validation
 const (
-	PostTitleMaxCharacters    = 280
+	// PostStatusMaxCharacters - Maximum length of the status field of a post
+	PostStatusMaxCharacters = 280
+	// PostLongFormMaxCharacters - Maximum length of the longForm field of a post
 	PostLongFormMaxCharacters = 50000
-	MaxPostTags               = 50
-	PostTagsMaxCharacters     = 80
+	// MaxPostTags - Maximum number of tags a post can have
+	MaxPostTags = 50
+	// MaxPostChannels - Maximum number of channels a post can be addressed to
+	MaxPostChannels = 30
+	// PostTagsMaxCharacters - Maximum character length of a tag
+	PostTagsMaxCharacters = 256
+	// PostChannelsMaxCharacters - Maximum character length of a channel
+	PostChannelsMaxCharacters = 256
+	// PostReferenceMaxCharacters - Maximum character length of a reference
+	PostReferenceMaxCharacters = 256
 )
 
+// Errors
+
+// ErrPostUnknownPanic - post has an unknown panic error
+var ErrPostUnknownPanic = errors.New("Unknown panic")
```

Comment:
@drwasho all these vars can be grouped together. 

```
var (
      // vars here
)
```

Reply:
Nice, didn't know that trick.